skymp5-front/src/components/SkyrimInput/index.js

```diff
@@ -4,14 +4,14 @@ import './styles.scss'
 
 const SkyrimInput = (props) => {
     let width = props.width != undefined ? props.width : 320;
-    let height = props.height != undefined ? props.height : 42;
+    let height = props.height != undefined ? props.height : 48;
     let text = props.text;
     let disabled = props.disabled;
     return (
         <div>
             <span className={'login-form--content_main__label___text'}>{text}</span>
             <div
-                className={`skymp-input`}
+                className={`skymp-input1`}
```

Comment:
I don't like this approach. We should not use vague classNames. Please update SkyrimButton className to something else and leave SkyrimInput className as 'skymp-input'